src/routes/Routes.tsx

```diff
@@ -145,6 +154,28 @@ const Routes = () => {
             hasBackLink: false,
           }}
         />
+        <WalletRequiredRoute
```

Comment:
I think we'll need to change this to a "AlwaysAccesible" route_(yes, I know, I told you to require a wallet in the specs)_, but now I'm thinking that the users should be able to browse it without having connected (and controls disabled in the component of course)

This is because these links will be shared publicly, meaning users will go directly to them, and users who didn't log into colony previously will be redirected to 404, which will not go over well

Reply:
That scenario makes sense! I agree that it's better to leave it as public as possible